tests/h/search/query_test.py

```diff
@@ -498,7 +498,7 @@ def storage(self, patch):
 
 class TestUriCombinedWildcardFilter():
 
-    @pytest.mark.parametrize('params,expected,separate_keys', [
+    @pytest.mark.parametrize('params,expected_ann_idxs,separate_keys', [
```

Comment:
idxs? Did you mean "ids"?

Reply:
I really did mean idxs as in indexes. I guess I'll change it to indexes spelled out since it was still unclear.